qa/rpc-tests/graphene_optimized.py

```diff
@@ -12,7 +12,17 @@
 class GrapheneOptimizedTest(GrapheneBlockTest):
     
     def setup_network(self, split=False):
-        node_opts = [
+        standard_node_opts = [
+            "-rpcservertimeout=0",
+            "-debug=graphene",
+            "-use-grapheneblocks=1",
+            "-use-thinblocks=0",
+            "-compute-optimize-graphene=0",
```

Comment:
could add -use-compactblocks=0 also.

Reply:
I assume that you mean to add `-use-compactblocks=0` to both configs (standard and optimized). Is that right?

Reply:
Done.